spec/docker-compose/docker_compose_spec.rb

```diff
@@ -244,6 +244,32 @@
     container1.stop
   end
 
+  it 'supports setting labels as an array' do
+    container1 = @compose.containers.values.first
```

Comment:
I suggest you to get the container by its label or name instead by its index on array, as show below:

```ruby
container1 = @compose.get_containers_by(label: 'busybox1').first
```

So, changing containers order on YAML file won't break the tests ;)
(There are some tests that still uses this positional approach, but their refactoring are in my roadmap)